api/tests/Feature/PoolCandidatesPaginatedTest.php

```diff
@@ -34,7 +34,9 @@ class PoolCandidatesPaginatedTest extends TestCase
 
     public PoolCandidate $applicantCandidate;
 
-    public string $query = /** GraphQL */ '
+    public string $query =
+    /** GraphQL */
+    '
```

Comment:
Everything looks good! Merge after fixing the linting error. This indentation is causing the linting error btw.

Reply:
fixed in [fix lint error](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11022/commits/3eff274b92b79fa05eb952ce106bd8d17a1beabc)